tools/vsimporter/xib2nib/UILabel.cpp

```diff
@@ -188,14 +184,12 @@ void UILabel::ConvertStaticMappings(NIBWriter* writer, XIBObject* obj) {
         AddInt(writer, "UIBaselineAdjustment", _baselineAdjustment);
     }
 
-    if (_numberOfLines == 1) {
-        if (_adjustsFontSizeToFit) {
-            AddBool(writer, "UIAdjustsFontSizeToFit", _adjustsFontSizeToFit);
-        }
-
-        if (_minimumFontSize != -1.0f) {
-            AddOutputMember(writer, "UIMinimumFontSize", new XIBObjectFloat(_minimumFontSize));
-        }
+    if (_minimumFontSize != -1.0f) {
+        AddOutputMember(writer, "UIMinimumFontSize", new XIBObjectFloat(_minimumFontSize));
+    } else if (_minimumScaleFactor != -1.0f) {
+        AddOutputMember(writer, "UIMinimumScaleFactor", new XIBObjectFloat(_minimumScaleFactor));
+    } else {
+        AddBool(writer, "UIAdjustsFontSizeToFit", false);
```

Comment:
>false [](start = 50, length = 5)

So we're changing it to always false in X2N (previous default was true). By design?